pkg/crc/machine/start.go

```diff
@@ -167,13 +171,17 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 			if err != nil {
 				return nil, errors.Wrap(err, "Cannot create cluster configuration")
 			}
+
+			telemetry.SetStartType(ctx, telemetry.AlreadyRunningStartType)
 			return &StartResult{
 				Status:         vmState,
 				ClusterConfig:  *clusterConfig,
 				KubeletStarted: true,
 			}, nil
 		}
 
+		telemetry.SetStartType(ctx, telemetry.StartStartType)
+
```

Comment:
I think it was better when it was done earlier in one of the previous iterations.